app/assets/javascripts/components/features/ui/containers/status_list_container.jsx

```diff
@@ -27,6 +27,9 @@ const makeGetStatusIds = () => createSelector([
       if (showStatus) {
         const regex = new RegExp(columnSettings.getIn(['regex', 'body']).trim(), 'i');
         showStatus = !regex.test(statusForId.get('reblog') ? statuses.getIn([statusForId.get('reblog'), 'unescaped_content']) : statusForId.get('unescaped_content'));
+        if (showStatus && statusForId.get('spoiler_text').length > 0) {
+          showStatus = !regex.test(statusForId.get('reblog') ? statuses.getIn([statusForId.get('reblog'), 'unescaped_spoiler_content']) : statusForId.get('unescaped_spoiler_content'));
```

Comment:
I was gonna suggest what @nightpool said, except just concatenate all of the searchable strings when creating `unescaped_content` in the reducer. In fact, I suggest renaming that field to "search_index" so it's more clear what it is used for